jena-arq/src/main/java/org/apache/jena/sparql/expr/aggregate/AggMedian.java

```diff
@@ -78,7 +82,7 @@ public boolean equals(Aggregator other, boolean bySyntax) {
         @Override
         protected void accumulate(NodeValue nv, Binding binding, FunctionEnv functionEnv)
         { 
-            if ( DEBUG ) System.out.println("median: "+nv) ;
+			log.debug("median  "+ nv);
 
```

Comment:
Style point: add `if ( log.isEnabled() )` otherwise `"median  "+ nv` is calculated regardless of loggingbe on or off.

Or use `("median {}", nv)`.

Reply:
I much prefer the latter, myself-- leave the logging logic to the logging framework! :grin:

Reply:
Ok I have added the preferred log style to Median and MedianDistinct. If this meets the requirements I can make changes at Avg, Min and Max accordingly as well.

Reply:
Sure, if you have time that would be very nice @neumarcx, thank you!